README.md

```diff
@@ -173,6 +173,33 @@ const styles = StyleSheet.create({
 
 Aphrodite will ensure that the global `@font-face` rule for this font is only inserted once, no matter how many times it's referenced.
 
+### Caveats
+
+#### Assigning a string to a content property for a pseudo-element
+
+When assigning a string to the `content` property it requires double quotes in CSS.
```

Comment:
True, made the changes to the text to reflect this and added in an extra example.